java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -816,10 +816,13 @@ public final ColumnVector normalizeNANsAndZeros() {
   /**
    * Create a deep copy of the column while replacing the null mask. The resultant null mask is the
    * bitwise merge of null masks in the columns given as arguments.
+   * The result will be sanitized to not contain any non-empty nulls in case of nested types
    *
    * @param mergeOp binary operator, currently only BITWISE_AND is supported.
```

Comment:
nit: Code says BITWISE_OR is supported, but docs to not.  (This is not your code, just noticed it)